ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/ledger/api/validation/CompletionServiceRequestValidatorTest.scala

```diff
@@ -37,18 +37,15 @@ class CompletionServiceRequestValidatorTest
 
   private val endReq = CompletionEndRequest(expectedLedgerId)
 
-  private val errorCodesVersionSwitcher_mock = mock[ErrorCodesVersionSwitcher]
   private val validator = new CompletionServiceRequestValidator(
     domain.LedgerId(expectedLedgerId),
     PartyNameChecker.AllowAllParties,
-    errorCodesVersionSwitcher = errorCodesVersionSwitcher_mock,
   )
 
-  val fixture = new ValidatorFixture((selfServiceErrorCodesEnabled: Boolean) => {
+  val fixture = new ValidatorFixture(() => {
```

Comment:
No need to have this fixture anymore

Reply:
01cb10f65cae5d9c087f4b585cefb22d762652dd